packages/medusa/src/api-v2/admin/invites/accept/route.ts

```diff
@@ -23,7 +24,17 @@ export const POST = async (req: MedusaRequest, res: MedusaResponse) => {
     user: req.validatedBody as AdminPostInvitesInviteAcceptReq,
   } as InviteWorkflow.AcceptInviteWorkflowInputDTO
 
-  const { result: users } = await workflow.run({ input })
+  let users
+  try {
+    const { result: userss } = await workflow.run({ input })
+    users = userss
+  } catch (e) {
+    if (e.message.startsWith("jwt malformed")) {
+      res.status(401).json({ message: "Unauthorized" })
+      return
+    }
```

Comment:
thought: I wonder if we should just always respond with 401 in case of errors to not leak information about the authentication?